### PR TITLE
Removal of charges panel

### PR DESCRIPTION
**Pull request by teryhill**
I think this gets them all #571

### CODE REVIEW

File: interface/patient_file/encounter/diagnosis.php [hunk 1]
```diff
@@ -1,312 +0,0 @@
-<?php
```
Comment (aethelwulffe) [inline interface/patient_file/encounter/diagnosis.php]: LibreEHR\interface\super\rules\library\RuleCriteriaDiagnosis.php may be calling this file.

File: library/globals.inc.php [hunk 2]
```diff
@@ -2635,7 +2629,7 @@
         'ms' => xl('Malay'),
         'ml' => xl('Malayalam'),
         'mt' => xl('Maltese'),
-        'mi' => xl('Māori'),
+        'mi' => xl('Maori'),
         'mr' => xl('Marathi (Marāṭhī)'),
```
Comment (aethelwulffe) [inline library/globals.inc.php]: If you are removing non-US keyboard characters, maybe you should do all of them before they get translated here?  I think this stuff should only be included when translations are on (period).

Comment (teryhill) [inline library/globals.inc.php]: These could be causing some of the collation errors

File: interface/patient_file/encounter/diagnosis.php (hunk 1 above)
Comment (teryhill) [inline interface/patient_file/encounter/diagnosis.php]: He has a return with that file name. I am not sure what he is doing, but it is not pointing to the interface/patient_file/encounter/ directory. 

Tell me do y'all want me to put it back?  @tmccormi @aethelwulffe

File: library/globals.inc.php (hunk 2 above)
Comment (aethelwulffe) [inline library/globals.inc.php]: Well, there are a lot more non-US keyboard characters in all that.  Any suggestions?

**Comment by aethelwulffe** [comment]
Looks like you got to replace the globals.inc.php

**Comment by teryhill** [comment]
Resolved

**Comment by aethelwulffe** [comment]
Mr. Tony,
Please look this over and give it a push if you would sir.

**Comment by aethelwulffe** [comment]
@tmccormi  I realize you are correct, but I think @teryhill intention was to fix a silent-fail issue.  There is some issue there that needs to be addressed, so we need to get that issue documented and hopefully replicated.

**Comment by tmccormi** [comment]
I do not think that this fixes the silent fail.  That would be database.sql not globals.inc.php.

**Comment by aethelwulffe** [comment]
I figured that the silent fail was when this stuff was dumped into the database, and I thought is wasn't included in database.sql...rather processed to do the inserts the first time it is called vs. within the actual schema file.  I forget how this particular mess works sometimes though...as does everyone else.

**Comment by tmccormi** [comment]
Yes, it does not relate to this issue.  It should be a separate issue and tested, not mixed in with a demolition commit.

**Comment by tmccormi** [comment]
Done.  thanks for pandering to my OCD about mixing issues ...

**Comment by aethelwulffe** [comment]
Sure thing, Ling-Ling!

**Pull request merged**